control/src/spitest.c

```diff
@@ -0,0 +1,134 @@
+/******************************************************************************
```

Comment:
This file shouldn't be here, it's just general example SPI code. Make a directory in sensors/ for workshop code like this so it doesn't clutter the main project area and prevent execution.
Also, nothing in src/ should have a main() function

Reply:
Fixed

Reply:
I have now "emptied" the files to allow for updates to test the spi base code